config/services.yaml

```diff
@@ -85,6 +85,10 @@ services:
             $databaseDir: '%security_advisories_db_dir%'
             $databaseRepo: '%security_advisories_db_repo%'
 
+    Buddy\Repman\Service\Telemetry:
+        arguments:
+            $instanceIdFile: '%kernel.project_dir%/var/instance-id'
```

Comment:
You can bind `$instanceIdFile` variable name to `'%kernel.project_dir%/var/instance-id'`. Then it will be injected automatically :mage_man: